<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Read_Zakaz</class>
 <widget class="QWidget" name="Read_Zakaz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Заказ кодов маркировки</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>631</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Выберите файл заказа:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="edtFilename"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnOpen">
        <property name="text">
         <string>Обзор</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListView" name="lstItems"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btnRead">
        <property name="text">
         <string>Разобрать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDel">
        <property name="text">
         <string>Удалить позицию</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="text">
         <string>Закрыть</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
